pom.xml

```diff
@@ -12,27 +12,35 @@
             <id>jitpack.io</id>
             <url>https://jitpack.io</url>
         </repository>
+        <repository>

```

Comment:
Why do we need this external repository?

Reply:
Let me check this in a clean environment(not my slt machine) because when I executed it, Maven couldn't download the jaxb-glassfish dependency.  Thanks!

Reply:
I've used the previous version of the jaxb dependency to remove the unnecessary staging repo.